pyproject.toml

```diff
@@ -19,7 +19,7 @@ requires = [
 
 [tool.flit.metadata.requires-extra]
 test = [
-    "mousebender",
+    "mousebender==v2.0.0",
```

Comment:
Why is this needed? Should we upgrade?

Reply:
`mousebender` API has changed quite a lot, and the used function does not exist anymore. Since this was use to parse the HTML response in tests, I thought this is better than refactoring the test more, the new API does not provide the same parsing result. Happy to dive deeper if you think it is useful.

Reply:
Okay, I have updated the test to use the latest mousebender.